### PR TITLE
Improve Mac OSX shared native makefile build

### PR DESCRIPTION
**Pull request by mgreter**
Needs https://github.com/sass/sassc/pull/256

- Use dylib instead of so on Mac OSX
- Remove unnessecary MSVC project file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -279,19 +291,27 @@ version: $(SASSC_BIN)
 test: test_build
 
 test_build: $(SASSC_BIN)
-	$(RUBY_BIN) $(SASS_SPEC_PATH)/sass-spec.rb -c $(SASSC_BIN) --impl libsass $(LOG_FLAGS) $(SASS_SPEC_PATH)/$(SASS_SPEC_SPEC_DIR)
+	$(RUBY_BIN) $(SASS_SPEC_PATH)/sass-spec.rb -c $(SASSC_BIN) --impl libsass \
+	--cmd-args "-I $(SASS_SPEC_PATH)/$(SASS_SPEC_SPEC_DIR)" \
```
Comment (nschonni) [inline Makefile]: the "-I" stuff was fixed upstream now, but it doesn't hurt to duplicate it

Comment (mgreter) [reply Makefile]: Seems I missed the sass-spec PR that fixed this!?

Comment (nschonni) [reply Makefile]: Yeah, it was https://github.com/sass/sass-spec/pull/1438. I landed it after I got one thumbs-up since it was blocking other PRs

Comment (mgreter) [reply Makefile]: Cool, glad it got fixed, but as you said, no harm in keeping the explicit option.

**Comment by mgreter** [comment]
Merged sassc counterpart, so this should go green after merging.

**Pull request merged**